### ISSUE TITLE
Mac OS X Download Fails

### ISSUE THREAD

**Issue by orsenthil** [CoatiSoftware/SourcetrailBugTracker#374]
Tried to download from https://www.sourcetrail.com/downloads On May 9, 2017 and I wasn't successful.

**Comment by egraether** [comment]
@orsenthil I just tried myself and it works fine for me. Which browser are you using? Are you on a corporate network that might monitor and block certain downloads? We once had an issue related to that: #273

**Comment by orsenthil** [comment]
I am using Chrome and I tried both on Corporate Network as well as Home Network.

Even today (May 16). It took me 10+ minutes to download SourceTrail 0.12.25 and clang-3-9-sourcetrail-.zip individually.

**Comment by orsenthil** [comment]
Didn't fail today, but the download was extremely slow. Should you use CDN to provide downloads?

**Comment by egraether** [comment]
You are the first one with issues so far. If we hear more cases then we will definitely look into it.

**Comment by orsenthil** [comment]
Sounds good. FWIW, I try it from SF Bay area.   We can close this issue and if I run into this again (frequently), I will update.

**Comment by egraether** [comment]
No, I would rather leave it open for a while. It might lead to more developers telling us about it.

**Comment by egraether** [comment]
+1, another report from the Bay Area via mail

**Comment by egraether** [comment]
+1, another report from the US

**Comment by haishibei** [comment]
I encountered the same issue: download is too slow and fails.
